### PR TITLE
PR For Stable Release of NTNU SmallSat HYPSO

### PR DESCRIPTION
**Pull request by DevAlvaroF**
Checklist
- [ x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ x] Source is from official source.
- [ x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x ] If static libraries are linked in, the license of the static library is packaged.
- [ x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ x] Build number is 0.
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by DevAlvaroF** [comment]
@conda-forge/help-python  Ready for review!

File: recipes/hypso/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    - pip
+    - setuptools >=61.0
+    - setuptools_scm  >=3.4
+  run:
+    - python >=3.9
+    - numpy >=1.24
+    - gdal =3.6.2
+    - cartopy >=0.21.1
+    - pandas
+    - matplotlib
```
Comment (ocefpaf) [inline recipes/hypso/meta.yaml:30]: Unless you explicitly needs qt consider using just matplotlib-base instead of the heavier package with the qt bindings.

```suggestion
    - matplotlib-base
```

Comment (DevAlvaroF) [reply recipes/hypso/meta.yaml]: I did the modifications in the meta.yaml and the pyproject.toml (https://github.com/NTNU-SmallSat-Lab/hypso-package/blob/v1.0.1/pyproject.toml) but got this error from the test in the meta.yml file. Any ideas? 

```
import: 'hypso'
import: 'hypso'
+ pip check
hypso 1.0.1 requires matplotlib-base, which is not installed.
```

Comment (ocefpaf) [reply recipes/hypso/meta.yaml]: @DevAlvaroF there are some misconceptions here you should not change the name in the Python package metadata. There is no such thing as "matplotlib-base" in the Python world.

Conda is an agnostic package manager and, in the world of diverse packages, we can break them down to make them easier to install. In this case, the python package named matplotlib was broken down into the "full matplotlib", that few people really needs it, and the matplotlib-base that removes the qt dependency making it easier to install, lighter, and taking less space. B/c virtually everyone uses notebooks theses days, installing qt to have matplotlib is not desired. 

TL;DR you don't need a new release or any change upstream, just here you need to change matplotlib to matplotlib-base to avoid download qt b/c you probably don't need it.

Comment (DevAlvaroF) [reply recipes/hypso/meta.yaml]: Thanks for the explanation! I learn something new every time. All tests passed <3
